### PR TITLE
make replicas configurable

### PR DESCRIPTION
**Pull request by johanneskastl**

### CODE REVIEW

File: charts/iperf3/values.yaml [hunk 1]
```diff
@@ -4,3 +4,6 @@ service:
   # externalIp:
   # the port to bind on the service, if enabled.
   port: 40000
+# uncomment and change this to set the number of replicas
+# default is 1 replica
+# replicas: 1
```
Comment (EugenMayer) [inline charts/iperf3/values.yaml:9]: please set the default in the values, not in the deployment via gotpl

File: charts/iperf3/templates/deployment.yml [hunk 2]
```diff
@@ -3,7 +3,7 @@ kind: Deployment
 metadata:
   name: {{.Chart.Name}}
 spec:
-  replicas: 1
+  replicas: {{ .Values.replicas | default 1 }}
```
Comment (EugenMayer) [inline charts/iperf3/templates/deployment.yml:6]: please set the default in the values, not in the deployment via gotpl

**Review by EugenMayer — CHANGES_REQUESTED** [review]
Just adjust the default placement and this one is good to go.

File: charts/iperf3/values.yaml (hunk 1 above)
Comment (johanneskastl) [reply charts/iperf3/values.yaml]: Done.

File: charts/iperf3/templates/deployment.yml (hunk 2 above)
Comment (johanneskastl) [reply charts/iperf3/templates/deployment.yml]: done

**Review by EugenMayer — APPROVED** [review]
Great, thanks! LGTM

**Comment by johanneskastl** [comment]
Conflicts fixed, is now mergeable into main.

**Comment by EugenMayer** [comment]
Thank you for staying at it!

**Pull request merged**